packages/contracts/deploy/migrations/1674773002__transfer_lendermanager_ownership_to_tellerv2.ts

```diff
@@ -0,0 +1,29 @@
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+import { migrate } from 'helpers/migration-helpers'
+import { LenderManager, TellerV2 } from 'types/typechain'
+
+const deployFn: DeployFunction = migrate(
+  __filename, // DO NOT CHANGE THIS LINE - it is used to determine the migration id
+  async (hre) => {
+    const tellerV2 = await hre.contracts.get<TellerV2>('TellerV2')
+    const lenderManager = await hre.contracts.get<LenderManager>(
+      'LenderManager'
+    )
+
+    if ((await lenderManager.owner()) !== tellerV2.address)
+      await lenderManager.transferOwnership(tellerV2.address)
+
+    if ((await tellerV2.lenderManager()) === hre.ethers.constants.AddressZero)
+      await tellerV2.setLenderManager(lenderManager.address)
+
+    if (await tellerV2.paused()) await tellerV2.unpauseProtocol()
```

Comment:
we can remove this as well - no longer necessary as we are not pausing the protocol to migrate active loans
```suggestion
```